src/ape_pm/compiler.py

```diff
@@ -39,6 +39,10 @@ def compile(
             ):
                 # Raw contract type JSON
                 contract_type_data = data
+                if "deployedBytecode" in data:
+                    contract_type_data["runtimeBytecode"] = {"bytecode": data["deployedBytecode"]}
+                if "bytecode" in data:
+                    contract_type_data["deploymentBytecode"] = {"bytecode": data["bytecode"]}
```

Comment:
How do your contract type data end up in this state?
Is it possible you can migrate it downstream somehow before compiling?

Reply:
It is possible, but I think its better if ape accepts the commonly used format by all other tools rather than adding another step to edit the artifact to make it "ape-compatible"

Reply:
Disregard this!